java-frontend/src/test/java/org/sonar/java/viewer/Viewer.java

```diff
@@ -139,38 +124,25 @@ private void checkForUpdate() {
     }
   }
 
-  private void analyse(String source) {
-    CFG cfg = buildCFG(source);
-    addCFGText(cfg);
-    String dot = CFGDebug.toDot(cfg);
-    WebEngine webEngine = webView.getEngine();
-    webEngine.executeScript("loadCfg('" + dot + "')");
-  }
-
-  private void addCFGText(CFG cfg) {
-    cfgText.setText(CFGDebug.toString(cfg));
+  protected void analyse(String source){
+//    new CFGViewer(this).analyse(source);
```

Comment:
Not convince it's the best approach. I don't want to have to comment one or the other in order to start the viewer.